docs/_articles/en/lwc/linting.md

```diff
@@ -11,37 +11,47 @@ Linting provides errors about malformed code while you edit. VS Code enforces Sa
 
 ## Installation
 
-1. If you created an SFDX project using the `force:project:create` command, the project may contain a `package.json` file with the ESLint plugin already included. Open `package.json` and verify that it contains the following:
+### For a New Project
 
-   ```json
-   "scripts": {
-       "lint": "npm run lint:lwc",
-       "lint:lwc": "eslint force-app/main/default/lwc"
-   }
-   ```
+If you create an SFDX project using the `sfdx force:project:create` command, your project contains a `package.json` file with the ESLint plugin already included.
 
-2. If your `package.json` does not contain the above lint commands, run `npm install @salesforce/eslint-config-lwc eslint@5 -D`.
+1. To install the ESLint plugin and other dependencies, run `npm install` in your project directory.
 
-3. To start linting, run `npm run lint:lwc`. You must have components in your project in order to run this command.
+2. Copy the [.eslintrc.json](https://github.com/forcedotcom/salesforcedx-templates/blob/master/src/templates/project/.eslintrc.json) file and save it to your project directory. Configure the ESLint plugin rules as needed.
```

Comment:
Link to configure ESLint plugin rules?
Also is it necessary to do? Could we add "optionally" somewhere?

Reply:
You mean link to the section? Sure, I can add that. I can change "Configure the ESLint plugin rules as needed" to "Configure the ESLint plugin rules (optional)."

Reply:
Cool